lib/web/JsonpMainTemplatePlugin.js

```diff
@@ -354,6 +354,34 @@ class JsonpMainTemplatePlugin {
 					return Template.asString([
 						source,
 						"",
+						"var rules = {",
+						Template.indent([
+							"createScriptURL: function(input) {",
+							Template.indent("return input;"),
+							"},",
+							"createURL: function(input) {",
+							Template.indent("return input;"),
+							"}"
+						]),
+						"}",
+						"var trustedTypesPolicy;",
+						"function getPolicy() {",
```

Comment:
Policy must be created eagerly. Otherwise it's possible for other scripts to claim the webpack assigned policy name before webpack can create it.

Reply:
This is not true. If a script claims webpack assigned policy, it means it has already control over the code and potentially can introduce more sever attacks. Also, I am not sure I understand which scripts are run before webpack. If I misunderstanding the comment can you please clarify what I am missing?